### ISSUE TITLE
What can generalist repositories do for you? A community feedback gathering session from the NIH Generalist Repository Ecosystem Initiative program

### ISSUE THREAD

**Issue by anavangulick** [ohbm/osr2024#5]
***By Ana Van Gulick, Figshare, NIH Generalist Repository Ecosystem Initiative program***

## Short description and the goals for the session
The neuroimaging community has been a leader in open science and data sharing for many years and neuroimaging researchers are frequent users of both discipline-specific and generalist data repositories as part of their open science workflows. On behalf of the NIH Generalist Repository Ecosystem Initiative (GREI), we propose this emergent session to learn from the neuroimaging community about how they use generalist repositories (GRs) for sharing data and other research materials and to gather feedback on how GREI could prioritize its work to enhance GR functionality and resources to better serve the needs of this research community. 

In February 2022, the US National Institutes of Health (NIH) Office of Data Science Strategy (ODSS) launched the [Generalist Repository Ecosystem Initiative](https://datascience.nih.gov/data-ecosystem/exploring-a-generalist-repository-for-nih-funded-data) (GREI), which brings together seven generalist repositories ([Dataverse](https://dataverse.org/), [Dryad](https://datadryad.org/stash), [Figshare](https://figshare.com/), [Mendeley Data](https://data.mendeley.com/), [Open Science Framework](https://osf.io/), [Vivli](https://vivli.org/), and [Zenodo](https://zenodo.org/)) to work collaboratively to enhance support for data sharing and discovery in GRs. GREI recognizes that GRs play a key role in the data sharing landscape for the FAIR sharing of data in trusted repositories, offering broad flexibility to publish any file type and any research output alongside discipline- and method-specific data repositories when they are available, especially for researchers seeking to comply with global data sharing mandates and to practice open science. Together the GREI repositories are working to enhance common metadata, persistent identifiers, and standard metrics to support cross repository search to lower the barriers for data sharing and reuse. 

In this emergent session we propose presenting a short overview of GREI goals and activities to enhance GR support for data sharing and discovery. We will then facilitate an interactive audience poll activity and audience discussion to learn about neuroimaging use cases for GRs including for sharing non-data materials and to uncover gaps in GR functionality and needs for resources or other support.

GREI would like to learn from OSR participants about their data sharing and repository experiences and hear from researchers what GRs could do to better support them through functionality or resources. Conducting community engagement with disciplinary research communities is a key objective for GREI that the program will use to inform our future work and we recognize that while many data repository resources are available in the neuroimaging community, the volume and diversity of research outputs to share necessitates the use of GRs for some outputs. Neuroscience is a top research category for all of the GREI repositories, often in part because these repositories are used to publish materials beyond data including software and code, images and media files, workflows, posters and presentations, and other supplementary files. GRs are also often used in conjunction with disciplinary repositories and data standards. We believe that as users of GRs and keen practitioners of open science with a wide variety of data types and research outputs to share, the neuroimaging community is an especially valuable group for GREI to engage with to inform our work. 

**Goals**

- Briefly present the objectives and outputs of the NIH Generalist Repository Ecosystem Initiative
- Conduct an interactive audience poll exercise to learn how neuroimaging researchers use GRs alongside discipline-specific data repositories to share data and other research materials 
- Facilitate an audience discussion to gather feedback about how generalist repositories could enhance their functionality and resources to better serve the needs of the neuroimaging community

## Useful Links
GREI Overview: [https://datascience.nih.gov/data-ecosystem/generalist-repository-ecosystem-initiative](https://datascience.nih.gov/data-ecosystem/generalist-repository-ecosystem-initiative)
GREI Outputs: [https://zenodo.org/communities/grei/](https://zenodo.org/communities/grei/)
GREI Blog: [https://medium.com/@blog-grei](https://medium.com/@blog-grei)

Tagging @anavangulick ```

**Comment by slugtmeijer** [comment]
Thank you for submitting such an elaborative and interesting topic! We would be happy to host this as an emergent session in the OSR. It will be either emergent session 4 or 5 (Thursday 26-06 15:45-16:45 or Friday 27-06 13:00-14:00), I'm just waiting to hear back from one of the other submissions for the schedule. Will someone be on site for the session? Feel free to reach out if there are any questions. I'll get back on which session it will be as soon as possible. 

Kind regards,
Selma

**Comment by anavangulick** [comment]
Thanks very much! Look forward to it! 

I think you mean **Wednesday** 26-06 and **Thursday** 27-06, yes? 
I will be on site in Seoul for the session, however due to travel Thursday afternoon it would be greatly appreciated if this session could be scheduled for the Wednesday 26-06 15:45-16:45 time if that is possible. Thanks!

**Comment by slugtmeijer** [comment]
Apologies, yes Wednesday or Thursday! I'll try for Wednesday. I'll hear back as soon as I know more.

**Comment by slugtmeijer** [comment]
It all worked out, you can have the Wednesday session! I'll send some more information when I've arrived in Seoul.

**Comment by anavangulick** [comment]
Fantastic! Thanks. See you in Seoul!

**Comment by slugtmeijer** [comment]
https://ohbm.github.io/osr2024/schedule/ 
There you find the corwdcast link for the session to share with people if you want.

**Comment by slugtmeijer** [comment]
Please if possible be at the OSR at 15:30 for the emergent session later today so we can set up and start in time. I'm looking forward to it.

**Comment by anavangulick** [comment]
Yes, will definitely come ahead to test the tech connections. Thanks!

**Comment by anavangulick** [comment]
Do I do the screen sharing of slides and poll via Crowdcast? The IT team here isn't able to help me
